.github/dependabot.yml

```diff
@@ -43,8 +43,8 @@ updates:
       prefix: "chore"
       include: "scope"
 
- # Android Intent Plus dependencies updates config
-      
+      # Android Intent Plus dependencies updates config
```

Comment:
Why all the different indentations?

Reply:
Because of autoformat

Reply:
Is it a blocker? And it is fixed already, by the way.